### PR TITLE
feat: [#175060536] new design delete profile

### PR DESCRIPTION
**Pull request by debiff**
## Short description
This PR handle the delete profile flow.

## List of changes proposed in this pull request

-  managed the profile deletion
-  added RadioButtonList core component
- managed the abort of the profile deletion

deleteProfile

### CODE REVIEW

File: ts/screens/profile/RemoveAccountDetailsScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+          },
+          shufflePinPadOnPayment
+        )
+      )
+  };
+};
+
+const mapStateToProps = (state: GlobalState) => {
+  const userDataProcessing = userDataProcessingSelector(state);
+  const isLoading =
```
Comment (Undermaken) [inline ts/screens/profile/RemoveAccountDetailsScreen.tsx:192]:
```suggestion
  const isLoading =
    pot.isLoading(userDataProcessing[UserDataProcessingChoiceEnum.DELETE]) ||
    pot.isUpdating(userDataProcessing[UserDataProcessingChoiceEnum.DELETE]);
```

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: ts/screens/profile/RemoveAccountDetailsScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+      headerTitle={I18n.t("profile.main.title")}
+    >
+      {props.isLoading ||
+      pot.isError(
+        props.userDataProcessing[UserDataProcessingChoiceEnum.DELETE]
+      ) ? (
+        <LoadingErrorComponent
+          isLoading={props.isLoading}
+          loadingCaption={loadingCaption}
+          onRetry={() => true}
```
Comment (Undermaken) [inline ts/screens/profile/RemoveAccountDetailsScreen.tsx:85]: ?

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: ts/components/RemovedAcoountInfoOverlay.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    zIndex: 1000,
+    backgroundColor: "#00C5CA",
+    width: widthPercentageToDP(100)
+  }
+});
+
+const RemovedAccountInfoOverlay: React.FunctionComponent = () => (
+  <View style={styles.versionContainer} pointerEvents="box-none">
+    <InfoBox iconColor={customVariables.brandDarkestGray}>
+      {/* <View style={{ paddingVertical: 15, paddingRight: 24 }}> */}
```
[the remark this answers is not in the document]
Comment (debiff) [reply ts/components/RemovedAcoountInfoOverlay.tsx:22]: @Undermaken This component is never used. We chosen to use the alert instead.
It's better if I remove it?

File: ts/components/core/selection/RadioButtonList.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  selectedItem?: T;
+  onPress: (selected: T) => void;
+};
+
+/**
+ * A list of radio button with an optional heading.
+ * The management of the selection is demanded and drived by the `selectedItem` prop.
+ * The item with the `id` equal to the `selectedItem` is the active one.
+ */
+export const RadioButtonList = <T,>(props: Props<T>) => {
```
[the remark this answers is not in the document]
Comment (debiff) [reply ts/components/core/selection/RadioButtonList.tsx:24]: I didn't get this.
Can you explain it please?

**Review by Undermaken — APPROVED** [review]

**Pull request merged**